README.md

```diff
@@ -27,32 +27,27 @@ com.unity.render-pipelines.high-definition-config | [![](https://badge-proxy.cds
 com.unity.shadergraph | [![](https://badge-proxy.cds.internal.unity3d.com/ad6f7b2b-97ec-46c5-8539-9b70e8c30bb5)](https://badges.cds.internal.unity3d.com/packages/com.unity.shadergraph/build-info?branch=master) [![](https://badge-proxy.cds.internal.unity3d.com/067b8f44-3f3a-4925-8462-996ffbe41662)](https://badges.cds.internal.unity3d.com/packages/com.unity.shadergraph/dependencies-info?branch=master) [![](https://badge-proxy.cds.internal.unity3d.com/7e1ee3c6-0477-4076-a2af-3376ead10421)](https://badges.cds.internal.unity3d.com/packages/com.unity.shadergraph/dependants-info) [![](https://badge-proxy.cds.internal.unity3d.com/9ce9cc97-b89d-4a2a-98c2-d1a1d2d0277e)](https://badges.cds.internal.unity3d.com/packages/com.unity.shadergraph/warnings-info?branch=master) ![ReleaseBadge](https://badge-proxy.cds.internal.unity3d.com/e2171d56-50c8-4803-964c-a63dcc728355) ![ReleaseBadge](https://badge-proxy.cds.internal.unity3d.com/30fe71f1-5838-4bf9-84eb-26a42320e4a2)
 com.unity.visualeffectgraph | [![](https://badge-proxy.cds.internal.unity3d.com/0fbfa6fc-2faf-4689-a3e7-fca736ab23cb)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/build-info?branch=master) [![](https://badge-proxy.cds.internal.unity3d.com/6606630d-31a9-4af5-b63c-25272411c381)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/dependencies-info?branch=master) [![](https://badge-proxy.cds.internal.unity3d.com/c10f50c2-2a79-4d0a-a763-54dcb40d027f)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/dependants-info) [![](https://badge-proxy.cds.internal.unity3d.com/823df233-071e-4ceb-a39f-b810d7fe6fe1)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/warnings-info?branch=master) ![ReleaseBadge](https://badge-proxy.cds.internal.unity3d.com/59b6ec9b-c477-4767-82ba-d2390e70cede) ![ReleaseBadge](https://badge-proxy.cds.internal.unity3d.com/ae2fb4f5-43dc-4ad2-8c94-7190dbcdc132)
 
-## Using the latest version
```

Comment:
This `using the latest version` section were outdated and described workflows related to package versioning which don't make sense in newer unity versions anymore. I moved the section at the bottom of the document and renamed it to  `## Package versions on Unity 2020.3 LTS and below`